src/util/basketUtils.js

```diff
@@ -77,3 +80,9 @@ export function setLendAmount({ amount, apollo, loanId }) {
 export function setDonationAmount() {
 	// TODO
 }
+
+export function handleInvalidBasket({ loan, cookieStore }) {
+	cookieStore.remove('kvbskt', { path: '/', secure: true });
+	cookieStore.set('failedLoan', JSON.stringify(loan));
```

Comment:
Could we please namespace this cookie a bit... maybe `kvatbid` or something like that...